mgnify_pipelines_toolkit/analysis/amplicon/make_asv_count_table.py

```diff
@@ -43,10 +43,13 @@ def parse_args():
 
 def order_df(taxa_df):
 
-    if len(taxa_df.columns) == 7:
-        taxa_df = taxa_df.sort_values(["Kingdom", "Phylum", "Class", "Order", "Family", "Genus"], ascending=True)
+    if len(taxa_df.columns) == 8:
+        taxa_df = taxa_df.sort_values(["Kingdom", "Phylum", "Class", "Order", "Family", "Genus", "Species"], ascending=True)
     elif len(taxa_df.columns) == 10:
         taxa_df = taxa_df.sort_values(["Domain", "Supergroup", "Division", "Subdivision", "Class", "Order", "Family", "Genus", "Species"], ascending=True)
+    else:
+        print("Data frame not the right size, something wrong.")
```

Comment:
Can you use the logging module, and use logger.error() here? That will help debugging this if this error happens.